src/main/java/org/asamk/signal/Manager.java

```diff
@@ -16,18 +16,70 @@
  */
 package org.asamk.signal;
 
-import com.fasterxml.jackson.annotation.JsonAutoDetect;
```

Comment:
Please avoid moving around the import statements. I suppose this was an automatic change from your IDE.
It makes the diff difficult to read and might cause unnecessary merge conflicts later.